docs/HTTP3.md

```diff
@@ -115,6 +115,7 @@ Build curl
 Build quiche and BoringSSL:
 
      % git clone --recursive https://github.com/cloudflare/quiche
+     % cd ..
```

Comment:
Shouldn't it then do `cd quiche` here and then `cd ..` *after* cargo is invoked?

Reply:
This should be `cd quiche`, all following steps expect it to be run from the build directory.
`cd ..` is already present in the next step ("Build curl").

Reply:
ah yes, even so!